streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/config/types/StateBackendType.java

```diff
@@ -0,0 +1,22 @@
+package io.ray.streaming.runtime.config.types;
+
+public enum StateBackendType {
```

Comment:
Is thers a state backend?

Reply:
The cencept is confused with streaming-state. Please use a different name

Reply:
changed to ContextBackendType